jetty-websocket/websocket-javax-server/src/main/config/modules/websocket-javax.mod

```diff
@@ -1,16 +1,20 @@
 DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
 
 [description]
-Enable websockets for deployed web applications
+Enable javax.websocket for deployed web applications.
+
+[tags]
+websocket
 
 [depend]
-# websocket client needs jetty-client
 client
-# javax.websocket needs annotations
 annotations
 
 [lib]
-lib/websocket/*.jar
+lib/websocket/websocket-core-${jetty.version}.jar
+lib/websocket/websocket-servlet-${jetty.version}.jar
+lib/websocket/jetty-javax-websocket-api-1.1.2.jar
```

Comment:
This seems a bit fragile, I can see this not being changed when the api version gets updated.

Reply:
I agree. There are few modules with the same issue...
I have started fixing that with 9.4.x (branch jetty-9.4.x-config_use_placeholder_for_version)
I was a bit conservative to make such big change.
But I can do it for 10.